mentat/config.py

```diff
@@ -146,6 +146,15 @@ class Config:
         },
     )
 
+    theme: str | None = attr.field(
+        default="light",
+        metadata={
+            "description": (
+                "Theme for interaction possible choices are light, dark or none."
+            )
+        },
```

Comment:
```suggestion
            ),
            validator=validators.in_(["light", "dark"]),
        },
```
You might import Themes here and use Themes.keys() instead, but idk if there's a circular dependency..

Reply:
Thought about it - was lazy. There should be no dependency - Themes are a simple set of dicts.